src/containers/BuySellCrypto/PaymentMethod/ManageWyreAccount/ManageWyrePersonalDetails.js

```diff
@@ -32,9 +36,25 @@ class ManageWyrePersonalDetails extends Component {
         dateOfBirth: null,
         socialSecurityNumber: null,
       },
+      date: new Date(),
+      mode: 'date',
+      toggleCalendar: false,
     };
   }
 
+  toggleCalendar = () => {
+    this.setState(prevState => ({
+      toggleCalendar: !prevState. toggleCalendar
+    }));
+  }
+
+  setDate = (event, date) => {
+    this.setState({
+      dateOfBirth: moment(date).format('YYYY-MM-DD'),
```

Comment:
Remove moment. We are not needding it as of now. Create helper to format your date using getDate, getMonth, getFullYear.